lib/taskana-core/src/main/java/pro/taskana/TaskQuery.java

```diff
@@ -215,10 +215,33 @@
      */
     TaskQuery customFieldsIn(String... customFields);
 
+    /**
+     * Calculating which elements should be returned for current list-page.<br>
+     * Setting the limit less then/or 0 will return all results without a limit. Offset of 0 or less will start at the
+     * first entry. The pageNumber does represent the offset and can cause a RuntimeException if it´s higher then the
+     * amount of result-rows.
+     *
+     * @param pageNumber
+     *            current page number which calculates the offset
+     * @param pageSize
+     *            max. amount of elements for this page
+     * @return the query
+     */
+    TaskQuery pagination(int pageNumber, int pageSize);
```

Comment:
Ich dachte, wir legen die Methode in den QueryMapper parallel zu list(offset, limit) als listPage(pageNo, pageSize). Und wir lenken das direkt im QueryMapper auf list(offset, limit) um. 
So, wie es jetzt ist, könnte ich sogar konkurrierend pagination angeben und dann list(offset, limit) aufrufen. Das macht ja keinen Sinn.

Reply:
Wird umgesetzt.
Wozu brauchen wir dann aber list(offset, limit) noch? Dann würde 1 Methode mit Begrenzung ausreichen denke ich.

Reply:
- BaseQuery hat listPage(pageNumber, pageSize) bekommen, die offset/limit berechnet und aufruft.
- Acc-Tests für alle Querys werden angepasst.
- list(offset, limit) wirft TaskanaRuntimeException, wenn das Offset "gesprengt" wird, statt Persistence (DB2) - H2 wirft keine. Ist Dokumentiert in den Testfällen.